src/System.Private.ServiceModel/src/System/IdentityModel/CryptoHelper.cs

```diff
@@ -76,6 +76,7 @@ internal static KeyedHashAlgorithm CreateKeyedHashAlgorithm(byte[] key, string a
             {
                 case SecurityAlgorithms.HmacSha1Signature:
 #pragma warning disable CA5350 // Do not use insecure cryptographic algorithm SHA1.
+                    // CodeQL [SM02200] We need to use HMACSHA1 as a requirement of SOAP protocols
```

Comment:
@imcarolwang my understanding is that the suppression needs to be on the same line, same for all fixes in this PR.